app/models/course/condition.rb

```diff
@@ -5,4 +5,12 @@ class Course::Condition < ActiveRecord::Base
   belongs_to :conditional, polymorphic: true
 
   delegate :satisfied_by?, to: :actable
+
+  def self.all_conditions
+    [
+      Course::Condition::Achievement,
```

Comment:
I suggest not passing around class objects because in development autoloading will not have these objects see changes.